gradle.properties

```diff
@@ -0,0 +1,4 @@
+org.gradle.caching = true
+org.gradle.parallel = true
+org.gradle.configureondemand = true
+org.gradle.vfs.watch = true
```

Comment:
If `org.gradle.vfs.watch=true` is default since gradle 7, then we already have it after update to gradle 7.2. I think we don't need to specify parameters enabled by default.

Reply:
You are right. `org.gradle.vfs.watch=true`   be removed